step.yml

```diff
@@ -0,0 +1,145 @@
+title: AppCenter Android Deploy
+summary: Deploy your awesome Android app to AppCenter.
+description: |-
+  Deploy an Android application to [AppCenter](https://appcenter.ms).
+
+  If an `app_id` is provided, the APK file will be uploaded to the specified app on HockeyApp. 
+  Otherwise HockeyApp will decide whether it's a new app or an update to an existing app based on the package ID.
+website: https://github.com/bitrise-steplib/steps-appcenter-android
+source_code_url: https://github.com/bitrise-steplib/steps-appcenter-android
+support_url: https://github.com/bitrise-steplib/steps-appcenter-android/issues
+
+project_type_tags:
+  - android
+  - xamarin
+  - cordova
+  - ionic
+  - react-native
+type_tags:
+  - deploy
+
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-hockeyapp-android-deploy
+
+inputs:
+  - apk_path: "$BITRISE_APK_PATH"
```

Comment:
Please use app_path and BITRISE_APP_PATH as we are migrating every step to use it. It can be an apk and aab too.

Reply:
@lszucs I will check if we can upload aab also. We will do the commented if the service is capable to deploy aabs

Reply:
added